lib/chef/provider/service/windows.rb

```diff
@@ -281,6 +266,21 @@ def action_configure_startup
 
   private
 
+  def configure_service_run_as_properties
+    return unless new_resource.property_is_set?(:run_as_user)
+
+    new_config = {
+      service_name: new_resource.service_name,
+      service_start_name: new_resource.run_as_user,
+      password: new_resource.run_as_password,
+    }.reject { |k, v| v.nil? || v.length == 0 }
+
+    Win32::Service.configure(new_config)
+    logger.info "#{new_resource} configured."
```

Comment:
We probably want to make this log the service_name property and not the resource name

Reply:
I agree. I'll update now.